tests/backends/base/test_creation.py

```diff
@@ -107,6 +108,18 @@ def test_migrate_test_setting_true(self, mocked_sync_apps, mocked_migrate, *mock
             with mock.patch.object(creation, '_destroy_test_db'):
                 creation.destroy_test_db(old_database_name, verbosity=0)
 
+    @mock.patch.dict(os.environ, {'RUNNING_DJANGOS_TEST_SUITE': ''})
+    @mock.patch.object(BaseDatabaseCreation, 'mark_expected_failures_and_skips')
+    def test_mark_expected_failures_and_skips_call(self, mark_expected_failures_and_skips, *mocked_objects):
+        test_connection = get_connection_copy()
```

Comment:
A docstring could be nice: "mark_expected_failures_and_skips() isn't called unless RUNNING_DJANGOS_TEST_SUITE is 'true'."